### ISSUE TITLE
iOS: eliminate non-styled UI like action sheets

### ISSUE THREAD

**Issue by Dejal** [samuelclay/NewsBlur#816]
As mentioned in #771, I think it'd be a good idea to eliminate other instances of UI that don't match the UI design of the app.  Generic UI like action sheets really glaringly stand out as different than the all-caps menus etc in the app's design language.

**Comment by nriley** [comment]
Definitely looks better — thanks.

**Comment by nriley** [comment]
Some stuff related to this, I think:

- nothing happens when you tap "Cancel" to close the add site popover

- popovers now always crash if I try to trigger them while they're already open (generally not possible from touch input, but this happens when I use the keyboard to do it).

```
2015-11-30 22:06:46.947 njrBlur[10331:494619] *** Terminating app due to uncaught exception 'NSGenericException', reason: 'UIPopoverPresentationController (<UIPopoverPresentationController: 0x7fd7dd140e10>) should have a non-nil sourceView or barButtonItem set before the presentation occurs.'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000109a45f45 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x00000001094bfdeb objc_exception_throw + 48
	2   UIKit                               0x0000000107f629ff -[UIPopoverPresentationController presentationTransitionWillBegin] + 3407
	3   UIKit                               0x0000000107723758 __71-[UIPresentationController _initViewHierarchyForPresentationSuperview:]_block_invoke + 2035
	4   UIKit                               0x000000010772138a __56-[UIPresentationController runTransitionForCurrentState]_block_invoke + 309
	5   UIKit                               0x00000001075c71ba _runAfterCACommitDeferredBlocks + 317
	6   UIKit                               0x00000001075da396 _cleanUpAfterCAFlushAndRunDeferredBlocks + 95
	7   UIKit                               0x00000001075e61c2 _afterCACommitHandler + 90
	8   CoreFoundation                      0x0000000109971947 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
	9   CoreFoundation                      0x00000001099718b7 __CFRunLoopDoObservers + 391
	10  CoreFoundation                      0x000000010996750b __CFRunLoopRun + 1147
	11  CoreFoundation                      0x0000000109966e08 CFRunLoopRunSpecific + 488
	12  GraphicsServices                    0x000000010af67ad2 GSEventRunModal + 161
	13  UIKit                               0x00000001075bb30d UIApplicationMain + 171
	14  njrBlur                             0x0000000105c6bb61 main + 65
	15  libdyld.dylib                       0x000000010a5a092d start + 1
)
```

One more thing which I'm not sure is related, so may need to report as a separate bug — tapping on an item from the "All Stories" list on the right side of the main screen on iPad no longer takes you to the article, it just takes you to the "All Stories" view.

**Comment by Dejal** [comment]
I'll fix those first two issues, unless you want to tackle them.

The last one sounds like #801 (selecting a story in All Stories doesn't always show the story), which I reported but later couldn't reproduce.  If you can reproduce it, please reopen and add steps to that issue.

**Comment by Dejal** [comment]
(Though those popover issues aren't really related to the changes for issue... but close enough.)

**Comment by nriley** [comment]
Yeah, I didn't bisect, they just don't happen in the release version as of last week :-)  Thanks!

**Comment by nriley** [comment]
Any progress on these? I have some time this weekend and Monday afternoon I could potentially devote to this, otherwise it may be the following weekend.

**Comment by Dejal** [comment]
I haven't done anything further on these yet, so you can do so if you wish.  I'm working on another issue, so I'll leave them to you for now; let me know if you don't want to. 

(Maybe we should use the assignee property to track who is working on what?  I'd be annoying if we accidentally duplicated effort on some issue.)

**Comment by nriley** [comment]
#801 is now fixed, and both of the remaining issues turned out to be the same bug and unrelated to this issue, so it's OK to close.  I agree having an assignee would be useful.

**Comment by samuelclay** [comment]
Ok @nriley, added you as a collaborator and assigned this ticket to you.

**Comment by nriley** [comment]
OK.  I was originally incorrect and the regressions turned out to be in #820, so I've merged my changes for that into the 5.1 branch.
